Cannot require('rollup-plugin-vue')

> node-sass, less and stylus are optional dependencies. If you are using scss/sass/less/stylus you should require (yarn add --dev node-sass less stylus) them.
> http://vuejs.github.io/rollup-plugin-vue/#/en/2.3/?id=style-languages

The document says so, but I encountered `Error: Cannot find module 'node-sass'` error even if I don't use sass.

### Expected behavior

No errors.

### Actual behavior

```
Error: Cannot find module 'node-sass'
    at Function.Module._resolveFilename (module.js:470:15)
    at Function.Module._load (module.js:418:25)
    at Module.require (module.js:498:17)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (****\test-
workspace\node_modules\rollup-plugin-vue\dist\rollup-plugin-vue.common.js:24:28)
    at Module._compile (module.js:571:32)
    at Object.Module._extensions..js (module.js:580:10)
    at Module.load (module.js:488:32)
    at tryModuleLoad (module.js:447:12)
    at Function.Module._load (module.js:439:3)
```

### Steps to reproduce the behavior

1. `$ npm install rollup-plugin-vue`
2. `$ node -e "require('rollup-plugin-vue')"`

--- a/src/script/coffee.js
+++ b/src/script/coffee.js
@@ -1,8 +1,6 @@
-import Compiler from 'coffeescript-compiler'
-
-const coffee = new Compiler()
-
 export default function (script) {
+    const Compiler = require('coffeescript-compiler')
+    const coffee = new Compiler()
     return new Promise((resolve, reject) => {
         coffee.compile(script.code, { bare: true }, (status, output) => {
             if (status === 0) {

--- a/src/style/less.js
+++ b/src/style/less.js
@@ -1,6 +1,5 @@
 export default async function (style, options) {
     const less = require('less')
-
     const { css, map } = await less.render(style.code, {
         sourceMap: {
             sourceMapFullFilename: style.id,

--- a/src/style/less.js
+++ b/src/style/less.js
@@ -1,6 +1,6 @@
-import less from 'less'
-
 export default async function (style, options) {
+    const less = require('less')
+
     const { css, map } = await less.render(style.code, {
         sourceMap: {
             sourceMapFullFilename: style.id,

--- a/src/style/scss.js
+++ b/src/style/scss.js
@@ -1,7 +1,7 @@
-import sass from 'node-sass'
 import debug from '../debug'
 
 export default function (style, options) {
+    const sass = require('node-sass')
     debug(`SASS: ${style.id}`)
     const { css, map } = sass.renderSync({
         file: style.id,

--- a/src/style/stylus.js
+++ b/src/style/stylus.js
@@ -1,6 +1,5 @@
-import stylus from 'stylus'
-
 export default async function (style, options) {
+    const stylus = require('stylus')
     const stylusObj = stylus(style.code, {...options.stylus})
         .set('filename', style.id)
         .set('sourcemap', {

--- a/src/template/pug.js
+++ b/src/template/pug.js
@@ -1,6 +1,5 @@
-import pug from 'pug'
-
 export default async function (template, extras, options) {
+    const pug = require('pug')
     const compiler = pug.compile(template, { filename: extras.id, ...options.pug })
 
     return compiler({css: extras.modules || {}})

